webui/react/src/pages/DesignKit.tsx

```diff
@@ -218,22 +226,117 @@ const DropdownsSection: React.FC = () => {
             { label: 'Option 3', value: 3 },
           ]}
         />
-        <strong>Disabled dropdown</strong>
-        <SelectFilter
+        <strong>Disabled Select</strong>
+        <Select
           defaultValue="disabled"
           disabled
           options={[{ label: 'Disabled', value: 'disabled' }]}
         />
-        <hr />
+        <strong>Variations</strong>
+        <strong>Select with search</strong>
+        <Select
+          filterOption={handleFilter}
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          placeholder="Search"
+        />
+        <strong>Multiple Select</strong>
+        <Select
+          disableTags
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          value={multiSelectValues}
+          onChange={(value) => setMultiSelectValues(value)}
+        />
+        <strong>Multiple Select with tags</strong>
+        <Select
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Select with tags and search</strong>
+        <Select
+          filterOption={handleFilter}
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          placeholder="Search"
+        />
+        <strong>Clearable Select</strong>
+        <Select
+          allowClear
+          disableTags
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          value={clearableSelectValues}
+          onChange={(value) => setClearableSelectValues(value)}
+        />
+        <strong>Select with sorted search results</strong>
+        <Select
+          disableTags
+          filterOption={(input, option) =>
+            (option?.label?.toString() ?? '').toLowerCase().includes(input.toLowerCase())
+          }
+          filterSort={(a: LabeledValue, b: LabeledValue) =>
+            (a?.label ? a.label : 0) > (b?.label ? b?.label : 0) ? 1 : -1
+          }
+          mode="multiple"
+          options={[
+            { label: 'Am', value: 1 },
+            { label: 'Az', value: 2 },
+            { label: 'Ac', value: 3 },
+            { label: 'Aa', value: 4 },
+          ]}
+          placeholder="Search"
+          value={sortedSelectValues}
+          onChange={(value) => setSortedSelectValues(value)}
+        />
+        <strong>Select with Max Tags and Placeholder</strong>
+        <Select
```

Comment:
I don't understand this variation. Aside from the custom placeholder it doesn't behave any different from a normal "multiple select".